src/AmazonS3EncryptionClientBase.cs

```diff
@@ -92,17 +92,10 @@ internal AmazonS3Client S3ClientForInstructionFile
 	    }
 
         internal AmazonS3CryptoConfigurationBase S3CryptoConfig { get; set; }
-#if BCL35
-        internal readonly Amazon.Extensions.S3.Encryption.Utils.ConcurrentDictionary<string, UploadPartEncryptionContext> CurrentMultiPartUploadKeys =
```

Comment:
Can the `Amazon.Extensions.S3.Encryption.Utils.ConcurrentDictionary` be removed since we targeting only versions of .NET that have the built in `System.Collections.Concurrent.ConcurrentDictionary`?

Reply:
Done